data_psi.go

```diff
@@ -23,6 +23,25 @@ const (
 	PSITableTypeTOT     = "TOT"
 	PSITableTypeUnknown = "Unknown"
 )
+const (
+	PSITableTypeIdPAT  = 0x00
```

Comment:
This would be handy as special type as well

Reply:
I like the idea of creating its own type as well

Reply:
done this as well